docs/commands/browser/runStep.mdx

```diff
@@ -0,0 +1,67 @@
+# runStep
+
+## Обзор {#overview}
+
+Используйте команду `runStep`, чтобы получить человекочитаемую историю выполнения теста, которая, в том числе, автоматически будет отображаться в [html-reporter][reporter].
+Шаги могут быть вложенными.
+
+## Использование {#usage}
+
+```javascript
+await browser.runStep(stepName, stepCb);
+```
+
+## Параметры команды {#parameters}
+
+<table>
+<thead>
+<tr><td>**Имя**</td><td>**Тип**</td><td>**Описание**</td></tr>
+</thead>
+<tbody>
+<tr><td>stepName</td><td>String</td><td>Название шага.</td></tr>
```

Comment:
Another thing we need to come to agreement about is formatting such tables with parameters.

I suggest to wrap param names in backticks and write types in TypeScript style, i.e.
`stepName`, `string`.
`stepCb`, `() => Promise<any>`

In my opinion it makes things a lot more clear and you can't express much with Javascript types "Object" or "Function". You have to way to express unions with JS types. And generally as far as I understand we're moving towards making TypeScript the primary language in Testplane